flags/src/main/java/org/secondbase/flags/Flags.java

```diff
@@ -363,13 +356,26 @@ private void addInstancedOption(
     }
 
     /**
+     * Pass args through secret handling.
+     * @param args - Arguments passed from main method.
+     * @return this
+     */
+    public String[] fetchSecrets(final String[] args) {
+        String[] ret = args;
+        for (final SecretHandler secretHandler : secretHandlers) {
+            ret = secretHandler.fetch(args);
+        }
+        return ret;
```

Comment:
You mean multiple SecretHandlers? It should call fetch(args) on all SecretHandler implementations, and the implementations themselves are responsible for fetching all secrets passed. I should add a dummy fetcher for testing inside of Flags component. And another test for multiple secret flags.